### PR TITLE
Create Cincinnati visiting.rst

### PR DESCRIPTION
**Pull request by hillaryfraley**

### CODE REVIEW

File: docs/include/conf/cincinnati/visiting.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Where to stay
+-------------
+
+We are not working with any of the hotels for attendee accommodations, but there are many hotels available within walking distance of the venue:
+
+- `Symphony Hotel <http://www.symphonyhotel.com>`_
+- `The Cincinnatian Hotel <http://www.cincinnatianhotel.com>`_
+- `21c Museum Hotel <https://www.21cmuseumhotels.com/cincinnati/>`_
+- `Millennium Cincinnati <https://www.millenniumhotels.com/en/cincinnati/millennium-hotel-cincinnati/>`_
+- `Hilton Cincinnati Netherland Plaza <http://www3.hilton.com/en/hotels/ohio/hilton-cincinnati-netherland-plaza-CVGNPHF/index.html>`_
```
Comment (shaunix) [inline docs/include/conf/cincinnati/visiting.rst]: Most of these are pretty pricey. Most affordable downtown hotels are probably Millenium and the new Holiday Inn. I've also directed people at the hotels in Newport if they're on a tight budget, and told them about the Southbank Shuttle.

File: docs/include/conf/cincinnati/visiting.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Taxi Companies
+~~~~~~~~~~~~~~
+
+Cincinnati has both Lyft and Uber. If you prefer local taxi companies, here is a list of `Cincinnati's licensed taxicab companies <https://www.cincinnati-oh.gov/public-services/linkservid/1831121A-DAD6-E032-9E01ECEF5B91DF25/showMeta/0/>`_.
+
+Please note that it is *not* common to hail taxis in Cincinnati. Your best bet is to call in advance.
+
+Driving to Union Hall
+~~~~~~~~~~~~~~~~~~~~~
+If you drive to the conference, parking in downtown Cincinnati is plentiful and inexpensive on the weekends. Use the `Over-the-Rhine Chamber of Commerce <http://otrchamber.com/pages/WheretoPark>`_ or `3CDC Parking <https://www.3cdc.org/where-to-park/>`_ sites to find parking facilities.
```
Comment (shaunix) [inline docs/include/conf/cincinnati/visiting.rst]: Probably worth pointing out the Mercer Garage is by far the closest, literally across the street.

File: docs/include/conf/cincinnati/visiting.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+How to get around
+-----------------
+
+Cincinnati is a small, walkable city with public transportation options available. Over-the-Rhine is about 1 mile north of the downtown area, which is bordered to the south by the Ohio River. We encourage you to connect with other documnentarians and explore during your stay.
+
+Streetcars and Buses
+~~~~~~~~~~~~~~~~~~~~
+
+Cincinnati has a `streetcar <http://www.cincinnatibellconnector.com>`_ that makes a loop from `Findlay Market <http://findlaymarket.org>`_ (north of Over-the-Rhine) to the `Great American Ball Park <https://www.mlb.com/reds/ballpark>`_ (downtown, on the Ohio River). You can also use Cincinnati's `bus network <http://www.go-metro.com/index.php>`_, which also connects to routes in northern Kentucky, just across the Ohio River.
```
Comment (shaunix) [inline docs/include/conf/cincinnati/visiting.rst:38]: Worth mentioning the two closest stops. 12th and Vine if coming from downtown. Washington Park if coming from the north.

**Review by shaunix — COMMENTED** [review]
This is great. Added some minor suggestions that don't need to block a merge.

File: docs/include/conf/cincinnati/visiting.rst (hunk 1 above)
Comment (hillaryfraley) [reply docs/include/conf/cincinnati/visiting.rst]: Thank you, Shaun! I added Holiday Inn, shuffled the list a bit, and added a note about hotels in Newport and the Southbank Shuttle.

File: docs/include/conf/cincinnati/visiting.rst (hunk 2 above)
Comment (hillaryfraley) [reply docs/include/conf/cincinnati/visiting.rst]: Thank you -- I added that info with a link to it on Google Maps.

File: docs/include/conf/cincinnati/visiting.rst (hunk 3 above)
Comment (hillaryfraley) [reply docs/include/conf/cincinnati/visiting.rst:38]: Thank you again! I added this information.

**Pull request merged**